awx/main/access.py

```diff
@@ -2288,11 +2275,6 @@ def filtered_queryset(self):
             Q(inventorysource__inventory__id__in=Inventory._accessible_pk_qs(
                 Inventory, self.user, 'read_role')))
 
-    def get_queryset(self):
-        # TODO: remove after the depreciation of v1 API
-        qs = super(UnifiedJobTemplateAccess, self).get_queryset()
-        return qs.exclude(inventorysource__source="")
```

Comment:
We should remove all inventory sources with `source=""` through a data migration, and remove empty string from the choices for the source field. (Django might help you auto-generate a migration file after you make the change to the choices)

I've been turning this over in my mind, and it just doesn't make any sense to keep them.